### PR TITLE
feat(AAPD-194): upload letters to neighbours

### PR DESCRIPTION
**Pull request by suityou01**
## Ticket Number

https://pins-ds.atlassian.net/browse/AAPD-194

## Description of change

Upload letters to neighbours

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js [hunk 1]
```diff
@@ -95,6 +95,12 @@ exports.questions = {
 		fieldName: 'letters-to-neighbours',
 		validators: [new RequiredValidator()]
 	}),
+	uploadLettersToNeighbours: new MultiFileUploadQuestion({
+		title: 'Upload the letters and emails',
```
Comment (chrisHowarthkc) [inline packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js:99]: The title here should be 'Uploaded letters' according to the Prototype

Comment (suityou01) [reply packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js:99]: But not according to the JIRA ticket.

**Review by chrishawldev — DISMISSED** [review]
Please make changes to reflect the prototype attached to the jira ticket and the feedback received from colleagues.

File: packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js (hunk 1 above)
Comment (suityou01) [reply packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js:99]: Agreed, but the prototype is secondary to the JIRA ticket as the JIRA ticket has the estimates, history, comment history and as such is the single source of truth and the project record of what was built. If there is a discrepancy between the ticket and the prototype then this should be resoved between the technical leadership team and the business. I will escalate this up to Helen for clarification.

File: packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js [hunk 2]
```diff
@@ -95,6 +95,12 @@ exports.questions = {
 		fieldName: 'letters-to-neighbours',
 		validators: [new RequiredValidator()]
 	}),
+	uploadLettersToNeighbours: new MultiFileUploadQuestion({
+		title: 'Uploaded letters',
+		question: 'Upload the letters and emails',
+		fieldName: 'upload-letters-emailss',
+		validators: [new RequiredValidator('You must add your documents')]
```
Comment (edpackard-work) [inline packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js:102]: for multifile validation, this line should now read:
`validators: [new RequiredFileUploadValidator(), new MultifileUploadValidator()]`

Comment (edpackard-work) [reply packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js:102]: (you won't need to specify the error message as it is the default for the RequiredFileUploadValidator)

File: packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js [hunk 3]
```diff
@@ -95,6 +95,12 @@ exports.questions = {
 		fieldName: 'letters-to-neighbours',
 		validators: [new RequiredValidator()]
 	}),
+	uploadLettersToNeighbours: new MultiFileUploadQuestion({
+		title: 'Uploaded letters',
+		question: 'Upload the letters and emails',
+		fieldName: 'upload-letters-emailss',
```
Comment (chrishawldev) [inline packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js:101]: We can now use the new html property from AAPD-301 in here. Main suggestion would be to use a naming convention like resources/upload-letters-emails/content.html for the location of the html

**Review by chrishawldev — APPROVED** [review]

**Pull request merged**